src/Microsoft.SqlTools.ServiceLayer/TableDesigner/TableDesignerService.cs

```diff
@@ -161,17 +161,17 @@ private Task HandleGenerateScriptRequest(TableInfo tableInfo, RequestContext<str
             });
         }
 
-        private Task HandleGeneratePreviewReportRequest(TableInfo tableInfo, RequestContext<GeneratePreviewReportResult> requestContext)
+        private Task HandleGeneratePreviewReportRequest(TableInfo tableInfo, RequestContext<PreviewReportResult> requestContext)
```

Comment:
Oh - so my comment in the other PR was just for the API stuff. This is fine as it was - having the result class be the name of the RPC method (`GeneratePreviewReport` in this case) + Result is the standard way of doing this.

So I don't think any changes here are necessary.

Reply:
Doesn't the API type name need to be the same in ADS and STS (I know it could work with different names because it's just json in the end), but different names would be an issue right? Also, we're the only ones consuming the Table Designer API, so doubt it would cause any confusion.

Reply:
No, it doesn't have to be the same at all. Just the shape of the object has to map (so name and type of properties inside)

Given your comment in the other thread I'd say just revert that back to match the name here to make it clear they're related.

Reply:
class/interface names do not matter when it comes to the communication between ADS and STS.

Reply:
i said the same thing above if you read it. But we why have different names when we are the only ones using it. Anyway, changing it back to `GeneratePreviewReportResult`